src/_locales/zh/messages.json

```diff
@@ -359,5 +359,29 @@
     "welcome_1b_dragging": "將備份檔案拖拽到此處…",
     "welcome_1b_file_selected": "已選取備份檔案：{{filename}}",
     "untitled_wallet": "未命名錢包",
-    "your_message": "您的留言…"
+    "your_message": "您的留言……",
+    "plugin_file_service_display_name": "插件: 文檔服務",
+    "plugin_file_service_file_key_copied": "已拷貝 FileKey",
+    "plugin_file_service_on_insert": "使用此文檔",
+    "plugin_file_service_uploading_preparing": "準備上傳……",
+    "plugin_file_service_uploading_estimating_time": "正在計算剩餘時間",
+    "plugin_file_service_uploading_in_minute_remaining": "剩餘 {{seconds}} 秒",
+    "plugin_file_service_uploading_remaining": "剩餘 {{minutes}} 分 {{seconds}} 秒",
+    "plugin_file_service_uploading_on_cancal": "上傳中，請勿關閉頁面",
+    "plugin_file_service_show_more": "更多",
+    "plugin_file_service_file_key": "File Key:",
```

Comment:
This field need not translate?

Reply:
CC @neruthes

Reply:
I believe that keeping English here is better, at least for now.